.changeset/afraid-waves-switch.md

```diff
@@ -0,0 +1,12 @@
+---
+"@aws-amplify/ui-react": patch
```

Comment:
Should be minor right? Also need @aws-amplify/ui package in here too

Reply:
Oops, updated that in latest commit.